3ds Max/Max2Babylon/Exporter/GlobalVertex.cs

```diff
@@ -19,9 +19,9 @@ public struct GlobalVertex
         public IPoint2 UV6 { get; set; }
         public IPoint2 UV7 { get; set; }
         public IPoint2 UV8 { get; set; }
-        public int BonesIndices { get; set; }
+        public int[] BonesIndices { get; set; }
```

Comment:
Maybe these should be `ushort` to begin with? It can't be bigger than this type right?